micro_ros_setup/scripts/build_firmware.sh

```diff
@@ -33,7 +33,12 @@ fi
 . $(dirname $0)/clean_env.sh
 
 # source dev_ws
-DEV_WS_DIR=$FW_TARGETDIR/dev_ws
+if [ $RTOS != "host" ]; then
```

Comment:
Maybe we can simply create an empty dev_ws for host builds, thus saving this kind of conditional.

Reply:
Already created in the last commits.